// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`E2E lint assertions-ref-required-error 1`] = `

validating /openapi.yaml...
[1] openapi.yaml:12:3 at #/paths/~1pets

PathItems should have refs

10 |   - url: http://petstore.swagger.io/v1
11 | paths:
12 |   /pets:
   |   ^^^^^
13 |     get:
14 |       summary: List all pets

Error was generated by the ref-required-no-property assertion rule.


[2] openapi.yaml:39:17 at #/paths/~1pets/get/responses/200/content/application~1json/schema

Response MediaType schema should have a ref

37 |     application/json:
38 |       schema:
39 |         properties:
   |         ^^^^^^^^^^^
40 |           id:
   |           ^^^
41 |             type: integer
   |             ^^^^^^^^^^^^^
42 | default:
43 |   description: unexpected error

Error was generated by the ref-required assertion rule.


[3] components/schemas/Owners.yaml:25:13 at #/get/responses/200/content/application~1json/schema

Response MediaType schema should have a ref

23 | application/json:
24 |   schema:
25 |     properties:
   |     ^^^^^^^^^^^
26 |       id:
   |       ^^^
27 |         type: integer
   |         ^^^^^^^^^^^^^

Error was generated by the ref-required assertion rule.


/openapi.yaml: validated in <test>ms

‚ùå Validation failed with 3 errors.
run \`openapi lint --generate-ignore-file\` to add all problems to the ignore file.


`;
